pkg/config/schema.go

```diff
@@ -20,11 +20,16 @@ var (
 		field.WithDescription("App client secret"),
 		field.WithRequired(true),
 	)
+	SyncPermissions = field.BoolField(
+		"sync-permissions",
+		field.WithDescription("Sync permissions"),
+	)
```

Comment:
Just an FYI, this is new, but @mindymo has indicated we should create a Jira issue for the public docs project whenever we add a new capability. https://conductorone.atlassian.net/jira/core/projects/DOCS/board?filter=&groupBy=status

Reply:
Created DOCS-370